build/builder.sh

```diff
@@ -134,4 +134,6 @@ docker run -i ${tty-} ${rm} \
   --env="CIRCLE_NODE_INDEX=${CIRCLE_NODE_INDEX-0}" \
   --env="CIRCLE_NODE_TOTAL=${CIRCLE_NODE_TOTAL-1}" \
   --env="GOTRACEBACK=${GOTRACEBACK-all}" \
+  --env="COVERALLS_TOKEN=${COVERALLS_TOKEN}" \
```

Comment:
If `COVERALLS_TOKEN` wasn't defined wouldn't that just become `--env="COVERALLS_TOKEN="`? Having a default empty string is fine, since the make command will throw an error message when it's empty.